core/src/main/java/org/elasticsearch/action/admin/indices/alias/IndicesAliasesRequest.java

```diff
@@ -61,7 +61,7 @@
 
     //indices options that require every specified index to exist, expand wildcards only to open indices and
     //don't allow that no indices are resolved from wildcard expressions
-    private static final IndicesOptions INDICES_OPTIONS = IndicesOptions.fromOptions(false, false, true, false);
+    private static final IndicesOptions INDICES_OPTIONS = IndicesOptions.fromOptions(false, false, true, false, true, false, true);
```

Comment:
Oh.. I was considering to leave the changes to `GetAliasesRequest` for another PR ( for better traceability ). But np, I will include them in this one.